chart/templates/api-deployment.yaml

```diff
@@ -49,7 +49,7 @@ spec:
             - name: POSTGRES_DATABASE
               value: '{{ .Values.api.env.postgresDatabase }}'
             - name: POSTGRES_SSL_DISABLED
-              value: '{{ .Values.api.env.postgresSslDisabled | default "false" }}'
+              value: '{{ .Values.api.env.postgresSslDisabled }}'
```

Comment:
makes sense that this was wrong given we were explicitly setting it to false

Reply:
yeah, to use "prefer" it needs to be `null` neither `false` nor `true`